src/components/form/text-input/index.tsx

```diff
@@ -61,28 +64,41 @@ export const TextInput = React.forwardRef<HTMLDivElement, TextInputProps>(
         iconBefore={iconBefore}
         {...wrapperProps}
       >
-        {title && (
-          <Text weight={theme.typography.weights.bold} size={theme.typography.sizes.m} className="inputTitle">
-            {title}
-          </Text>
-        )}
-        {subTitle && (
-          <Text size={theme.typography.sizes.xs} className="inputSubtitle">
-            {subTitle}
-          </Text>
-        )}
+        <div className='titleContainer'>
+          <div className={`${subtitle ? 'mb-4' : ''}`}>
+            {title && (
+              <Text weight={theme.typography.weights.bold} size={theme.typography.sizes.m} className='inputTitle'>
+                {title}
+              </Text>
+            )}
+
+            {
+              subtitle && (
+                <Text size={theme.typography.sizes.xs} className='inputSubtitle inputSubtitleLeft'>
+                  {subtitle}
+                </Text>
+              )
+            }
+          </div>
+          <div className='mb-4'>
```

Comment:
do we need empty `div` in case where `rightSubtitle` is missing?
@doichev-kostia

Reply:
Yep, otherwise the layout will be displayed badly

Reply:
Can it be redone somehow without empty div? @doichev-kostia 

divs allow flow content and empty divs are something generally to be avoided. You can read more [here](https://html.spec.whatwg.org/multipage/dom.html#palpable-content) about it.

Reply:
@lazzystep, Thanks for an article)
I've fixed the layout issue :)